eq-author/src/components/RichTextEditor/index.js

```diff
@@ -348,21 +378,79 @@ class RichTextEditor extends React.Component {
       : this.hasInlineStyle(editorState, style);
   };
 
+  state = { show: false, text: "", multiline: false };
```

Comment:
The show value can be renamed to describe the content being shown, e.g. showPasteModal

Reply:
Updated